Dockerfile

```diff
@@ -1,12 +1,13 @@
 FROM sameersbn/ubuntu:14.04.20170503
 MAINTAINER sameer@damagehead.com
 
-ENV GITLAB_VERSION=9.2.7 \
+ENV GITLAB_VERSION=9.3.0 \
     RUBY_VERSION=2.3 \
-    GOLANG_VERSION=1.6.3 \
-    GITLAB_SHELL_VERSION=5.0.4 \
-    GITLAB_WORKHORSE_VERSION=2.0.0 \
-    GITLAB_PAGES_VERSION=0.4.2 \
+    GOLANG_VERSION=1.8.3 \
```

Comment:
@solidnerd Separate ci or separate pr? but what for? These version updates are required by GitLab CE.